src/test/java/seedu/address/model/tag/TagTest.java

```diff
@@ -1,10 +1,14 @@
 package seedu.address.model.tag;
 
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static seedu.address.testutil.Assert.assertThrows;
 
 import org.junit.jupiter.api.Test;
 
 public class TagTest {
+    final String validTagName = "PM";
```

Comment:
Perhaps you could make use of the `VALID_TAG_PM` in the `CommandTestUtil.java` file